libs/SalesforceSDK/src/com/salesforce/androidsdk/util/test/TestCredentials.java

```diff
@@ -24,15 +24,20 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-package com.salesforce.androidsdk.smartsync;
+package com.salesforce.androidsdk.util.test;
```

Comment:
One one TestCredentials.java file living in the util.test package.
It reads the credentials from test_credentials.json.
test_credentials.json is under shared/test, but we have links to it in the assets folder of every test project.